tests/bluetooth/ctrl_llcp/src/main.c

```diff
@@ -431,11 +431,11 @@ typedef enum {
 	LL_ENC_RSP,
 	LL_START_ENC_REQ,
 	LL_START_ENC_RSP,
-} helper_opcode_t;
+} helper_pdu_opcode_t;
```

Comment:
Note extra comma at end of previous line

Reply:
Common construct found all over the code base